Code/Framework/AzToolsFramework/Tests/FocusMode/EditorFocusModeTests.cpp

```diff
@@ -31,18 +31,58 @@ namespace UnitTest
         EXPECT_EQ(m_focusModeInterface->GetFocusRoot(m_editorEntityContextId), AZ::EntityId());
     }
 
+    TEST_F(EditorFocusModeFixture, GetFocusedEntitiesRoot)
+    {
+        AzToolsFramework::EntityIdList entities = m_focusModeInterface->GetFocusedEntities(m_editorEntityContextId);
+        
+        using ::testing::UnorderedElementsAre;
+        EXPECT_EQ(entities.size(), 6);
```

Comment:
nit: The size check is now technically redundant as you get that for free with the `UnorderedElementsAre` check, if the number != 6, it will fail

Reply:
I was wondering if that could be the case, but I could not find confirmation after a quick search :D Good to know!